sources/packages/backend/apps/db-migrations/src/sql/NotificationMessages/Insert-offering-change-request-inprogress-with-student-message.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO
+  sims.notification_messages(id, description, template_id)
+VALUES
+  (
+    13,
+    'Application Offering Change Request In Progress With Student',
```

Comment:
I would say the message type could be "Institution request a change"

Reply:
I have requested the message from @JasonCTang as well since it might be shown to the users someday.

Reply:
Confirmed with business area that the wording is good to go!